application/init.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
+
+STATUS=0
+
+echo "tfc-backup: init: Started"
+
+start=$(date +%s)
+/usr/local/bin/restic init || STATUS=$?
```

Comment:
just a thought, and I'm not sure if it's even available in alpine, but it would be a lot simpler to use `time` to report the execution time, e.g. `time /usr/local/bin/restic init || STATUS=$?`. And come to think of it, do you really need to specify the /usr/local/bin path?

Reply:
`time` gives you four lines of output and I didn't want to parse the output.  I'm only including the the execution time in case a future user is trying to discover why the overall backup is taking so long.

I tend to include the path to executable programs when they aren't included in `/bin` or `/usr/bin`.  I've seen too many Unix-like systems that didn't include `/usr/local/bin` in the default `PATH` for all users.